daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -179,299 +179,6 @@ class EngineTest
     }
   }
 
-  "command translation" should {
```

Comment:
Why did you delete all of these?

Reply:
Let me rebase once  #10927 is merged. I think it will be more clear.

Reply:
Is it better now ?

Reply:
yes, thanks!